src/components/fileDownloadCard.tsx

```diff
@@ -63,6 +66,9 @@ const FileDownloadCard: React.FC<FileDownloadCardProps> = ({
                             {docName}
                         </Link>
                     </Text>
+                    <Text color={colourTheme.colors.Gray}>
+                        Date Submitted {convertToShortDateString(docUploadDate)}
+                    </Text>
```

Comment:
```suggestion
                    <Text color={colourTheme.colors.Gray}>
                        Date Submitted: {convertToShortDateString(docUploadDate)}
                    </Text>
```

Hmm, I think this looks a bit off here. Do you mind hacking the css props a bit so we can get this to be a single line?
We ideally want this: 
![image](https://user-images.githubusercontent.com/44826218/145328003-e1f8c926-7b26-4136-931c-51bd55046cc8.png)